application/models/policydependent.go

```diff
@@ -55,6 +56,24 @@ func (p *PolicyDependent) Update(tx *pop.Connection) error {
 	return update(tx, p)
 }
 
+func (p *PolicyDependent) Destroy(tx *pop.Connection) error {
+	return destroy(tx, p)
+}
+
+// canBeDeleted checks for related items
+func (p *PolicyDependent) RelatedItemNames(tx *pop.Connection) []string {
```

Comment:
I think it would be better to use RelatedItems() rather than just names, as we'll want more than just names in emails.

```suggestion
func (p *PolicyDependent) RelatedItems(tx *pop.Connection) []Items {
```